src/static/js/modules/simple-table-row-links.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+// Required for IE8 support.
+require( './polyfill/event-listener.js' );
+require( './polyfill/query-selector.js' );
+
+/**
+ * Makes a table row whose first table cell child is a link become the link
+ * for the entire table row.
+ */
+function init() {
+  var rows = document.querySelectorAll( '.simple-table__row-links tr' );
+  for ( var i = rows.length - 1; i >= 0; i-- ) {
+    _listenForRowClick( rows[i] );
+  }
+}
+
+/**
+ * Add a listener to a row.
+ *
+ * @param {HTMLElement} row A tr element in a table.
+ */
+function _listenForRowClick( row ) {
+  // The anonymous function is needed since IE8
+  // can't find the event's currentTarget otherwise.
+  row.addEventListener( 'mousedown', function() {
```

Comment:
hm, I see what you mean. @schaferjh do you have an opinion on this? Basically on `click` or `mouseup` the link will trigger when releasing the mouse button, on `mousedown` it will trigger when first pressing the mouse button over the link.